lib/logstash/inputs/rabbitmq.rb

```diff
@@ -81,7 +80,8 @@ class LogStash::Inputs::RabbitMQ < LogStash::Inputs::Threadable
   # Passive queue creation? Useful for checking queue existance without modifying server state
   config :passive, :validate => :boolean, :default => false
 
-
+  # Include amqp headers as k/v pairs
+  config :include_headers, :validate => :boolean, :default => false
```

Comment:
I like the idea to be able to add the `headers` from rabbitmq!
I think we should mimic what we are doing in the elasticsearch's input see https://github.com/logstash-plugins/logstash-input-elasticsearch/blob/master/lib/logstash/inputs/elasticsearch.rb#L87

1. I think we can drop the `include_` part and just use `headers`
2. Maybe add an example here with a sample of the headers that will be added.
3. Add an option to specify where to save the header and we should save it by default in the `@metadata` field see the elasticsearch input for an example.